### PR TITLE
fix: auth module

### PR DESCRIPTION
**Pull request by wtrocki**
Add example for authentication module.

### CODE REVIEW

**Comment by wtrocki** [comment]
I will be adding extra commit to remove rhoas cli token right now

File: examples/README.md [hunk 1]
```diff
@@ -1,69 +1,72 @@
...
-For more information about authentication please refer to SDK docs.
-- Repository is properly initiatilized: 
-
-```shell
-go mod tidy
-```
+## Prerequisites
 
+1. Go to  <https://console.redhat.com/openshift/token> and press "Load token".
+2. Copy token fro clipboard to file
```
Comment (dimakis) [inline examples/README.md:6]: from
```suggestion
2. Copy token from clipboard to file
```

File: examples/README.md [hunk 2]
```diff
@@ -1,69 +1,72 @@
...
 
-### For example:
+### For example
 
-### For kafkamgmt: 
+### For kafkamgmt
 
 ```shell
-ACCESS_TOKEN=`rhoas authtoken` go run ./examples/kafkamgmt/kafka_mgmt.go
+go run ./examples/kafkamgmt/kafka_mgmt.go
```
Comment (dimakis) [inline examples/README.md:18]: should we be using 
```bash
OFFLINE_TOKEN=`ocm token` go run ./examples/kafkamgmt/kafka_mgmt.go
```
here, this is the same token but expires quicker? unless I have this wrong (which I may do 😆 )

**Review by dimakis — COMMENTED** [review]
just some simple comments, LGTM

File: examples/README.md (hunk 2 above)
Comment (wtrocki) [reply examples/README.md:18]: This token never expires

**Pull request merged**